helm/oauth2-proxy/templates/servicemonitor.yaml

```diff
@@ -10,10 +10,7 @@ metadata:
 {{- end }}
   labels:
     app: {{ template "oauth2-proxy.name" . }}
```

Comment:
Why do you keep this one?

Reply:
The only reason I see is to keep the chart backward compatible with installations using this label.
At least we can add a note to the release because it is not a real breaking change.